server/app/controllers/me-controller.js

```diff
@@ -76,7 +76,6 @@ async function updateMe(req, res) {
     });
   } else if (user.barman) {
     const schema = BarmanSchema
-      .forbiddenKeys('leaveAt')
```

Comment:
This means that a barman could change his status anytime he wants

Reply:
Yes !

Reply:
@Embraser01 ?